examples/text_classification/pretrained_models/README.md

```diff
@@ -239,7 +239,7 @@ python -u deploy/serving/export_servable_model.py \
 在服务器端容器中，启动server
 
 ```shell
-python -m deploy/serving/paddle_serving_server_gpu.serve \
+python -m paddle_serving_server.serve \
```

Comment:
`paddle_serving_server_gpu` has been deprecated since v0.6.0 in Paddle Serving. 

So please also update the installation instruction if you update the command, thanks!

1) Update `paddle-serving-server-gpu==0.5.0` 
to `paddle-serving-server-gpu`

2) Update `https://github.com/PaddlePaddle/Serving/tree/v0.5.0` to `https://github.com/PaddlePaddle/Serving/tree/v0.6.0`

Reply:
Thanks! This explains the reason for the stale content.

Done in the new commit.

Please let me know if it is better to squash the three commits into one for clean git history.

Reply:
We will do the squash-merge after approved, no worries :)